lib/slack.js

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
TODO:

Should I break this down even further? Seems weird to have WebClient and Webhook stuff in here; maybe slack.emoji.js and slack.webhook.js?